On upstream inclusion of the assumed_charset patch (past 1.5.14), file_charset
got renamed to attach_charset. We add it back for compatibility.

Index: mutt/init.h
===================================================================
--- mutt.orig/init.h	2007-03-12 19:11:53.000000000 +0100
+++ mutt/init.h	2007-03-12 19:13:56.000000000 +0100
@@ -248,6 +248,9 @@ struct option_t MuttVars[] = {
   ** Note: "iso-2022-*" must be put at the head of the value as shown above
   ** if included.
   */
+  { "file_charset",	DT_SYN,  R_NONE, UL "attach_charset", 0 },
+  /*
+  */
   { "attach_format",	DT_STR,  R_NONE, UL &AttachFormat, UL "%u%D%I %t%4n %T%.40d%> [%.7m/%.10M, %.6e%?C?, %C?, %s] " },
   /*
   ** .pp
